pkg/schema/converter/crd.go

```diff
@@ -73,18 +75,6 @@ func forVersion(crd *v1.CustomResourceDefinition, group, kind string, version v1
 		attributes.SetColumns(schema, versionColumns)
 	}
 	if version.Schema != nil && version.Schema.OpenAPIV3Schema != nil {
-		if fieldsSchema := modelV3ToSchema(id, crd.Spec.Versions[0].Schema.OpenAPIV3Schema, schemasMap); fieldsSchema != nil {
-			for k, v := range staticFields {
-				fieldsSchema.ResourceFields[k] = v
-			}
-			for k, v := range fieldsSchema.ResourceFields {
-				if schema.ResourceFields == nil {
-					schema.ResourceFields = map[string]schemas.Field{}
-				}
-				if _, ok := schema.ResourceFields[k]; !ok {
-					schema.ResourceFields[k] = v
-				}
-			}
-		}
+		schema.Description = version.Schema.OpenAPIV3Schema.Description
```

Comment:
That's a valid comment. I can remove it from here.

Reply:
I also think it would be worth it to add a comment here, wasted a bit of time figuring out why we were updating description in both places until I found this thread.